### ISSUE TITLE
Keyboard shortcut to access the repeat count setting doesn't work anymore

### ISSUE THREAD

**Issue by dbitouze** [RMPR/atbswp#59]
## Checklist

- [x] I use the latest release of atbswp
- [x] The issue is not existing yet

## Verbose log

```python
#!/bin/env python3
# Created by atbswp (https://github.com/rmpr/atbswp)
# on 05 Nov 2020 
import pyautogui
import time
pyautogui.FAILSAFE = False

pyautogui.moveTo(1087, 812)
pyautogui.mouseDown(565, 342, 'left')
pyautogui.mouseUp(565, 342, 'left')
time.sleep(2)
pyautogui.mouseDown(906, 338, 'left')
pyautogui.mouseUp(906, 338, 'left')
time.sleep(1)
pyautogui.mouseDown(829, 498, 'left')
pyautogui.mouseUp(829, 498, 'left')
time.sleep(1)
pyautogui.mouseDown(867, 651, 'left')
pyautogui.mouseUp(867, 651, 'left')
time.sleep(2)
pyautogui.mouseDown(941, 200, 'left')
pyautogui.mouseUp(941, 200, 'left')
time.sleep(1)
```

### Operating System
Linux Mageia 8

### Desktop Environment/Window Manager
Plasma

### Python version
Python 3.8.9

## Description

The <kbd>Ctrl</kbd>+<kbd>R</kbd> keyboard shortcut to access the repeat count setting, that used to work nicely with `atbswp` 0.2, doesn't work with `atbswp` 0.3.0.

**Comment by RMPR** [comment]
Thanks for reporting, this commit
https://github.com/RMPR/atbswp/commit/55852d349b52a7d58ee2b9e4b004bc5213082500 
should fix that. You are welcomed to test, if everything is fine it will
most likely be available in atbswp 0.3.1

**Comment by dbitouze** [comment]
> You are welcomed to test

Seems to work nicely. And, ice on the cake, the <kbd>F11</kbd> seems to work again, too: thanks!
